apps/lottie-viewer/src/locations/Field.test.tsx

```diff
@@ -47,6 +45,6 @@ describe('Field', () => {
   it('renders LottiePreviewField with empty object if getValue is null', () => {
     mockSdk.field.getValue.mockReturnValue(null);
     render(<Field />);
-    expect(screen.getByTestId('lottie-preview-field')).toContain({});
+    expect(screen.getByTestId('lottie-preview-field').textContent).toBe('{}');
```

Comment:
After I updated the vite/vitest versions this test started failing, I think due to some stricter matching logic. It didn't like an empty object even though our app literally is wanting to check that the value for the field is an empty object. I guess a unique problem due to the nature of this app.